.mvn/maven.config

```diff
@@ -2,5 +2,6 @@
 --errors
 --update-snapshots
 --no-transfer-progress
--Dmaven.test.error.ignore=true -Dmaven.test.failure.ignore=true
```

Comment:
The properties maven.test.error/failure.ignore should be passed in Jenkins then in order to have simpler distinctions from build failures. Maybe for the GH Workflow too?

Reply:
No, the worksflows are configured in a way that they always run the post actions.